it/mail/chrome/messenger/mime.properties

```diff
@@ -264,7 +264,7 @@
 # LOCALIZATION NOTE (1038): Translate the following two lines as a single sentence. In the middle of the two sections
 # there will be a URL. You may translate the text in any order you wish, but the html tags must stay in the same locations.
 # In particular, the "<B>" tag must begin the first section, which must end with the "<A HREF=" tag. Do not translate the html tag,"<P>"
-1038=<B>Questo messaggio eccede l'impostazione di dimensione massima stabilita in "impostazioni account". Sono state pertanto scaricate dal server solamente le prime righe.<P>Fare clic <A HREF="
+1038=<B>Questo messaggio supera l'impostazione di dimensioni massime stabilita in "impostazioni account". Sono state pertanto scaricate dal server solamente le prime righe.<P>Fare clic <A HREF="
```

Comment:
CORRECTION NEEDED. 

Please replace "Fare clic" with "Clicca su". See line no. 300 below for reference.